trustedApplications/trustedApplicationsPane.ts

```diff
@@ -92,28 +109,41 @@ function createApplicationTable(subject) {
     createText('th', 'Access modes'),
     createText('th', 'Actions')
   ])
-  applicationsTable.appendChild(header)
+  applicationsTable.appendChild(header);
 
   // creating rows
-  kb.each(subject, ns.acl('trustedApp'))
-    .flatMap(app => kb.each(app, ns.acl('origin')).map(origin => ({appModes: kb.each(app, ns.acl('mode')), origin})))
-    .sort(({origin: a}, {origin: b}) => a.value < b.value ? -1 : 1)
-    .forEach(({appModes, origin}) => applicationsTable.appendChild(createApplicationEntry(subject, origin, appModes, updateTable)))
+  (kb.each(subject, ns.acl('trustedApp'), undefined, undefined) as any)
+    .flatMap((app: any) => {
+      return kb.each(app, ns.acl('origin'), undefined, undefined)
+        .map(origin => ({appModes: kb.each(app, ns.acl('mode'), undefined, undefined), origin}))
+    })
+    .sort((a: any, b: any) => a.origin.value < b.origin.value ? -1 : 1)
+    .forEach(({appModes, origin}: {appModes: NamedNode[], origin: NamedNode}) => applicationsTable.appendChild(createApplicationEntry(subject, origin, appModes, updateTable)))
 
   // adding a row for new applications
   applicationsTable.appendChild(createApplicationEntry(subject, null, [ns.acl('Read')], updateTable))
 
   return applicationsTable
 
   function updateTable() {
-    applicationsTable.parentElement.replaceChild(createApplicationTable(subject), applicationsTable)
+    applicationsTable.parentElement!.replaceChild(createApplicationTable(subject), applicationsTable)
```

Comment:
Nice, didn't know of [the non-null assertion operator in TS](https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#non-null-assertion-operator), always fun to learn new stuff ^_^

Reply:
:) I prefer not to use it, but since it worked without a non-null check so far, I figured it'd be OK to keep it that way.